packages/frontend/components/Sidebar/Sidebar.tsx

```diff
@@ -247,12 +253,19 @@ const Sidebar: React.FC<SidebarProps> = memo(
 
 interface NoMajorSidebarProps {
   selectedPlan: PlanModel<string>;
+  transferCourses: ScheduleCourse2<unknown>[];
 }
 
 export const NoMajorSidebar: React.FC<NoMajorSidebarProps> = ({
   selectedPlan,
+  transferCourses,
 }) => {
-  const creditsTaken = totalCreditsInSchedule(selectedPlan.schedule);
+  const transferCredits = transferCourses.reduce(
```

Comment:
does the regular sidebar (the one that is not "no major") already support adding transfer course credits? if not, we should abstract the logic and add to both locations.

Reply:
Yes, I believe lines 193 - 199 add to the creditsTaken variable for the normal sidebar.